pkg/user/logic.go

```diff
@@ -39,16 +39,39 @@ func (b *BusinessLogic) GetCatalog(w http.ResponseWriter, r *http.Request) (*osb
 	data := `
 ---
 services:
-- name: skeleton-example-service
```

Comment:
Outside of this PR range, but would it make more sense to read these from a yaml file instead?

Reply:
Absolutely it would